types/jsonwebtoken/v7/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+    "compilerOptions": {
+        "module": "node16",
+        "lib": [
+            "es6"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": true,
+        "strictFunctionTypes": true,
+        "types": [],
+        "noEmit": true,
+        "forceConsistentCasingInFileNames": true
```

Comment:
📝 Difference to https://github.com/DefinitelyTyped/DefinitelyTyped/pull/68516/files, in this version we don't need `exactOptionalPropertyTypes` option enabled.